launch/start.launch

```diff
@@ -33,10 +33,13 @@
                 -s $(env ROBOT_BRINGUP_PATH)/parameters/hardware/actuators/rgb_lights.ops
                 "/>
                 <arg name="NAME" value ="hardware"/>
-                <arg name="LOG_LEVEL" value="Info"/>
+                <arg name="LOG_LEVEL" value="Warning"/>
                 <arg name="DEBUG" value="$(arg debug)"/>
                 <arg name="machine" value="amigo1" />
             </include>
+            
+            <!-- BODYPART RESETTER ROBOCUP HACK -->
+            <node name="spindle_resetter" pkg="robot_launch_files" type="bodypart_resetter.py" args="amigo spindle 1"/>
```

Comment:
This is ok to leave in. Can be removed when https://github.com/tue-robotics/tue_robocup/issues/257 is solved